app/models/hostgroup.rb

```diff
@@ -110,7 +110,7 @@ def self.search_by_config_group(key, operator, value)
     property :operatingsystem, 'Operatingsystem', desc: 'Returns operating system to be used on hosts within this host group'
     property :os, 'Operatingsystem', desc: 'Returns operating system to be used on hosts within this host group'
     property :ptable, 'Ptable', desc: 'Returns partition table associated with this host group'
-    property :puppetmaster, String, desc: 'Returns host name of the server with Puppet Master'
+    property :puppet_server, String, desc: 'Returns host name of the server with Puppet Master'
```

Comment:
Perhaps a chance to update the description as well?
```suggestion
    property :puppet_server, String, desc: 'Returns host name of the server with Puppetserver
```